deploy/kiali/kiali_cr.yaml

```diff
@@ -642,6 +642,26 @@ spec:
 # Flag to indicate Kiali to enable/disable an Action to label a namespace for automatic Istio Sidecar injection.
 #    ---
 #    istio_injection_action: true
+#
+# Defines defaults for the UI
+#    ---
+#    ui_defaults:
+#
+# How long should the interval be between each graph refresh. Omit for default,
+# "pause" to disable auto refresh.
+# Valid values: pause, 10s, 15s, 30s, 1m, 5m, 15m
+#        ---
+#        auto_refresh_interval: "15s"
```

Comment:
This setting applies to more than just the graph.  It applies, I think, to every page offering auto refresh.  Maybe just change to:

` The automatic refresh interval for pages offering automatic refresh.`

Also, I think "automatic" is implied, so you could opt to remove "auto_".